src/cloudant/document.py

```diff
@@ -17,13 +17,19 @@
 """
 import json
 import posixpath
-import urllib
 import requests
-
 from requests.exceptions import HTTPError
 
+# pylint: disable=wrong-import-order
```

Comment:
agreed, all PY2 conditional imports have been moved to _2to3 module as suggested.